changedetectionio/model/Watch.py

```diff
@@ -39,6 +39,7 @@ class model(dict):
             'notification_muted': False,
             'notification_title': None,
             'notification_urls': [],  # List of URLs to add to the notification Queue (Usually AppRise)
+            'notification_screenshot': False,  # List of URLs to add to the notification Queue (Usually AppRise)
```

Comment:
`# List of URLs to add to the notification Queue (Usually AppRise)`

This could be a better comment :)